src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -156,12 +156,25 @@ public void visitToken(DetailAST ast) {
 
     /**
      * Checks whether a method has a javadoc comment.
+     *
```

Comment:
remove this change.

Reply:
Sorry, it is intellij idea formatting, I will remove it